include/wx/osx/iphone/private.h

```diff
@@ -225,13 +230,12 @@ protected :
 
     @end
 
-    @interface wxUIView : UIView
+    @interface wxUIView : UIScrollView
```

Comment:
I'm not sure what is `wxUIView` used exactly for, but it looks weird to make all views scrollable. Are you sure we need to do it?